src/Raven.Client/Documents/Session/AsyncDocumentSession.Stream.cs

```diff
@@ -152,24 +153,21 @@ public Task<IAsyncEnumerator<StreamResult<T>>> StreamAsync<T>(IQueryable<T> quer
             return StreamAsync(indexQuery, token);
         }
 
-        public Task<IAsyncEnumerator<StreamResult<T>>> StreamAsync<T>(long? fromEtag, int start = 0,
-                                                                    int pageSize = Int32.MaxValue, string transformer = null, Dictionary<string, object> transformerParameters = null, CancellationToken token = default(CancellationToken))
+        public Task<IAsyncEnumerator<StreamResult<T>>> StreamAsync<T>(string fromChangeVector, int start = 0, int pageSize = int.MaxValue, string transformer = null, Dictionary<string, object> transformerParameters = null, CancellationToken token = default(CancellationToken))
```

Comment:
How much sense does this make now? @ppekrol  @arekpalinski  Thoughts about removing it entirely?